index.html

```diff
@@ -183,29 +260,130 @@ <h4>What your wealth says about your health</h4>
+                                        <a target="_blank"
+                                            href="https://www.rollingstone.com/music/music-features/el-dorado-arkansas-musicfest-murphy-arts-district-706972/">
                                             View on Rolling Stone <i class="fas fa-external-link-alt"></i>
                                         </a>
                                     </p>
                                     <p>
-                                        <a target="_blank" href="/assets/El Dorado_RS_article copy.pdf" alt="">View PDF <i class="fas fa-external-link-alt"></i></a>
+                                        <a target="_blank" href="/assets/El Dorado_RS_article copy.pdf" alt="">View PDF
+                                            <i class="fas fa-external-link-alt"></i></a>
+                                    </p>
+                                </div>
+                            </figcaption>
+                        </figure>
+                    </li>
+                    <li data-aos="fade-up">
+
+                        <figure>
+                            <img src="/assets/PDFS/future islands.png" alt="">
+
+                            <figcaption>
+                                <div class='caption'>
+                                    <h4>
+                                        Future Islands: The Unlikely Rise of Baltimore’s Heartache Kings
+                                    </h4>
+                                    <p>
+                                        <a target="_blank"
+                                            href="https://www.rollingstone.com/music/music-features/future-islands-the-unlikely-rise-of-baltimores-heartache-kings-123082/">
+                                            View on Rolling Stone <i class="fas fa-external-link-alt"></i>
+                                        </a>
                                     </p>
+
+                                </div>
+                            </figcaption>
+                        </figure>
+                    </li>
+                    <li data-aos="fade-up">
+
+                        <figure>
+                            <img src="/assets/PDFS/shaw.png" alt="">
+
+                            <figcaption>
+                                <div class='caption'>
+                                    <h4>
+                                        Shannon Shaw's Southern Reckoning
+                                    </h4>
+                                    <p>
+                                        <a target="_blank"
+                                            href="https://www.rollingstone.com/music/music-features/shannon-shaw-nashville-auerbach-interview-698409/">
+                                            View on Rolling Stone<i class="fas fa-external-link-alt"></i>
```

Comment:
missed a space here between Stone and the `<i>` tag
(minor)

```suggestion
                                            View on Rolling Stone <i class="fas fa-external-link-alt"></i>
```

Reply:
Fixed this for you in a new branch

Reply:
thank you!